elixir/apps/web/lib/web/live/sites/index.ex

```diff
@@ -212,18 +229,68 @@ defmodule Web.Sites.Index do
         </div>
       </:title>
 
+      <:action>
+        <.docs_action path="/deploy/resources" fragment="the-internet-resource" />
+      </:action>
+
       <:action :if={Domain.Accounts.internet_resource_enabled?(@account)}>
-        <.edit_button navigate={~p"/#{@account}/sites/#{internet_gateway_group}"}>
-          Manage Full-route Tunneling
+        <.edit_button navigate={~p"/#{@account}/sites/#{@internet_gateway_group}"}>
+          Manage Internet Site
         </.edit_button>
       </:action>
 
       <:help>
-        This is a dedicated Site for Internet traffic that does not match any specific Resource.
-        Deploy Gateways here to secure access to the public Internet for your workforce.
+        Use the Internet Site to manage secure, private access to the public internet for your workforce.
       </:help>
 
-      <:content></:content>
+      <:content :if={
+        Domain.Accounts.internet_resource_enabled?(@account) &&
+          needs_internet_resource_migration?(@internet_resource, @internet_gateway_group)
+      }>
+        <div class="px-1 text-neutral-500">
+          <p class="mb-2">
+            ACTION REQUIRED: Please migrate your existing Internet Resource to this Site before <strong>March 15, 2025</strong>.
+          </p>
+          <p class="mb-8 text-sm">
+            <.website_link path="/blog/internet-resource-migration">
+              Read more about why this is necessary.
+            </.website_link>
+          </p>
+          <.button_with_confirmation
+            id="migrate_internet_resource"
+            style="warning"
+            confirm_style="warning"
+            icon="hero-exclamation-triangle-solid"
+            on_confirm="migrate_internet_resource"
+          >
+            <:dialog_title>
+              Confirm Internet Resource Migration from {@existing_internet_resource_group_name}
+            </:dialog_title>
+            <:dialog_content>
+              <p class="text-center my-8">
+                <.icon name="hero-exclamation-triangle-solid" class="w-16 h-16 text-primary-500" />
+              </p>
+              <p class="mb-2">
+                Migrating the Internet Resource will permanently
+                move it from the <strong>{@existing_internet_resource_group_name}</strong>
+                Site to the <strong>Internet</strong>
+                Site. This cannot be reversed.
+              </p>
+              <p>
+                Any Clients connected to this Resource will be immediately disconnected. You will need to deploy new Gateways in the Internet Site
+                to reconnect them.
+              </p>
```

Comment:
Another wording suggestion.

Here's how it will look:

<img width="666" alt="Screenshot 2025-02-14 at 6 52 09 PM" src="https://github.com/user-attachments/assets/a648f10d-094a-4497-9eb3-2634d2925e3c" />

```suggestion
              <p class="mb-2">
                Any Clients connected to this Resource will be immediately disconnected.
              </p>
              <p>
                To minimize downtime, it is recommended to deploy new Gateways in the Internet Site before completing the migration of the Internet Resource.
              </p>
```